src/molgenis/capice_resources/threshold_calculator/plotter.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from matplotlib import pyplot as plt
+
+from molgenis.capice_resources.core import GlobalEnums as Genums
+from molgenis.capice_resources.threshold_calculator import ThresholdEnums as Menums
+
+
+class ThresholdPlotter:
+    def __init__(self, recall_data: pd.DataFrame):
+        """
+        Main plotter function of ThresholdCalculator.
+
+        Initializes the best scores according to the first threshold (from 0 to 1) that hits a
+        recall between 0.94 and 0.96.
```

Comment:
Does this class have this knowledge? If not (might only be known by Calculator?), preferably remove the exact thresholds here and refer to the class that 'knows' this. Sadly enough for Python no class-links in pydoc is yet supported in IntelliJ: https://youtrack.jetbrains.com/issue/PY-27635